students/AnastasiaPletser/hw-1/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+  <p>I'm just a text</p>  
+  <script> alert('One option') </script>
+ <script src="./index.js"> </script> 
```

Comment:
You also can use auto-formatter here. It'll make your codding process more easer =)